### PR TITLE
feat(messaging): Added `hu` and `it` into message system

### LINKED ISSUE #9537: Add Hungarian and Italian to message system config
**Describe the bug**
We added new languages but did not extend message system config

### PR DESCRIPTION
**Pull request by hynek-jina**
Should resolve: https://github.com/trezor/trezor-suite/issues/9537

I've used Deepl for messgaes translations and some of them doublecheck with Google Transalate

### CODE REVIEW

File: suite-common/message-system/config/config.v1.json [hunk 1]
```diff
@@ -452,7 +468,9 @@
                     "es": "¡PRECAUCIÓN: ¡El firmware detectado no es oficial! Es posible que su Trezor sea falso. Póngase en contacto de inmediato con help@trezor.io.",
                     "cs": "UPOZORNĚNÍ: Byl detekován neoficiální firmware! Váš Trezor může být padělaný. Okamžitě kontaktujte help@trezor.io.",
                     "ru": "ВНИМАНИЕ: Обнаружено неофициальное ПО! Ваш Trezor может быть подделкой. Свяжитесь с нами немедленно на help@trezor.io.",
-                    "ja": "注意：非公式のファームウェアが検出されました！お使いのTrezorは偽物かもしれません。直ちにhelp@trezor.ioに連絡してください。"
+                    "ja": "注意：非公式のファームウェアが検出されました！お使いのTrezorは偽物かもしれません。直ちにhelp@trezor.ioに連絡してください。",
+                    "hu": "FIGYELEM: Nem hivatalos firmware-t észleltek! Lehet, hogy a Trezorod hamisítvány. Azonnal lépjen kapcsolatba a help@trezor.io címen.",
+                    "it": "CAUTION: Unofficial firmware detected! Your Trezor may be counterfeit. Contact help@trezor.io immediately."
```
Comment (tomasklim) [inline suite-common/message-system/config/config.v1.json]: Signore, not italiano

**Review by tomasklim — DISMISSED** [review]

File: suite-common/message-system/schema/config.schema.v1.json [hunk 2]
```diff
@@ -363,7 +363,7 @@
             "title": "Localization",
             "description": "A multilingual text localization.",
             "type": "object",
-            "required": ["en-GB", "en", "es", "cs", "ru", "ja"],
+            "required": ["en-GB", "en", "es", "cs", "ru", "ja", "hu", "it"],
```
Comment (matejkriz) [inline suite-common/message-system/schema/config.schema.v1.json]: maybe we should leave the new languages as optional, so we have a chance to simply remove or not use them in the future? What do you think, @tomasklim?

File: suite-common/message-system/config/config.v1.json [hunk 3]
```diff
@@ -572,7 +596,9 @@
                         "es": "Actualizar el Firmware",
                         "cs": "Aktualizovat Firmware",
                         "ru": "Обновить прошивку",
-                        "ja": "ファームウェアのアップデート"
+                        "ja": "ファームウェアのアップデート",
+                        "hu": "",
+                        "it": ""
```
Comment (matejkriz) [inline suite-common/message-system/config/config.v1.json]: todo

File: suite-common/message-system/schema/config.schema.v1.json (hunk 2 above)
Comment (tomasklim) [reply suite-common/message-system/schema/config.schema.v1.json]: Maybe official languages should be required and community can be optional?

File: suite-common/message-system/config/config.v1.json (hunk 1 above)
Comment (hynek-jina) [reply suite-common/message-system/config/config.v1.json]: Thx, updated

File: suite-common/message-system/config/config.v1.json (hunk 3 above)
Comment (hynek-jina) [reply suite-common/message-system/config/config.v1.json]: thanks, added

File: suite-common/message-system/schema/config.schema.v1.json (hunk 2 above)
Comment (matejkriz) [reply suite-common/message-system/schema/config.schema.v1.json]: that's what I thought too, but I would definitely leave what we already have marked as required, just mark new as required only if it become officially supported

**Review by matejkriz — APPROVED** [review]

**Pull request merged**